src/main/java/uk/gov/hmcts/reform/document/DocumentUploadClientApi.java

```diff
@@ -47,14 +50,34 @@ public DocumentUploadClientApi(
         this.objectMapper = objectMapper;
     }
 
-    public UploadResponse upload(
+    public UploadResponse uploadWithUserId(
         @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
         @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuth,
         @RequestHeader(USER_ID) String userId,
         @RequestPart List<MultipartFile> files
+    ) {
+        return upload(authorisation, serviceAuth, userId, Collections.emptyList(), files);
+    }
+
+    public UploadResponse uploadWithRoles(
+        @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation,
+        @RequestHeader(SERVICE_AUTHORIZATION) String serviceAuth,
+        @RequestHeader(USER_ID) String userId,
+        @RequestHeader(ROLES) List<String> roles,
```

Comment:
Roles is not a header .. it is a parameter in the DM end point

Reply:
adding these annotations to the method does nothing?
they should all be removed?